counterparty-core/counterpartycore/lib/api/compose.py

```diff
@@ -442,13 +444,18 @@ def compose_fairminter(
     :param lock_quantity: If True, the quantity of the asset cannot be changed after the minting
     :param divisible: If True, the asset is divisible
     :param description: The description of the asset. Overrides the current description if the asset already exists.
+    :param price: alias for `lot_price`
+    :param quantity_by_price: alias for `lot_size`
     """
+    price_arg = lot_price if lot_price > 0 else price
+    quantity_by_price_arg = lot_size if lot_size > 1 else quantity_by_price
+
     params = {
         "source": address,
         "asset": asset,
         "asset_parent": asset_parent,
-        "price": price,
-        "quantity_by_price": quantity_by_price,
+        "price": price_arg,
```

Comment:
can we easily support both the old and the new names?

Reply:
sure! that already the case, I just move the old params:
```
:param price: alias for `lot_price`
:param quantity_by_price: alias for `lot_size`
```